config/routes.rb

```diff
@@ -172,16 +169,34 @@
     get :sample_locations, on: :collection
   end
 
-  authenticate :user, ->(u) { u.admin? } do
-    mount RESQUE_SERVER, at: "/resque"
-  end
+  # TODO: - David - Check how to do that without devise
+  # authenticate :user, ->(u) { u.admin? } do
+  #   mount RESQUE_SERVER, at: "/resque"
```

Comment:
hmm this is kind of difficult because operationally I think we'll need the Resque dashboards to monitor and debug? unless this is pretty temporary? we could also do a check on @chanzuckerberg email perhaps to avoid the role fetch..

Reply:
I should have fixed that code before submitting the PR. I forgot to review these TODO. Fixing this code now.